### DISCUSSION TITLE
deSEC A record subname lack of causing issues

### DISCUSSION THREAD

**Discussion by cittris** [qdm12/ddns-updater#670] [Q&A]
Hi all,

First thanks to the dev for providing this tool. 
I've got my DNS with deSEC set up and working. My A record does not contain a value. If i try and enter "@" it errors and will not allow it. 
However a blank subname in DDNS-Updater defaults to "@" and the two are not talking to each other.

Is there something i'm doing incorrectly here or is there another way to approach this issue?

![Screenshot 2024-03-12 194400](https://github.com/qdm12/ddns-updater/assets/74881272/9091ca90-d9b3-4bff-9eb6-3372b3b348d1)
![Screenshot 2024-03-12 194316](https://github.com/qdm12/ddns-updater/assets/74881272/f5301c56-d754-4286-96ae-11ff59019073)
![Screenshot 2024-03-12 193542](https://github.com/qdm12/ddns-updater/assets/74881272/f9741595-8b6c-4fd5-97b3-7fa624f5241e)

Cheers,

**Comment by qdm12** [comment]
> If i try and enter "@" it errors and will not allow it.

What error do you get? desec allows you to set the host as `"@"`. It also defaults to `"@"` if you leave it unset.

>  and the two are not talking to each other.

Not sure what you mean here, ddns-updater and the registrar update server?

What logs do you get from the program? It seems that the IP address for yourdomain.com is the one matching your public IP address, so that can be a reason why no update request is done 🤔
